cupboard_app/test_api.py

```diff
@@ -1480,3 +1481,118 @@ def test_create_user_api_without_email_in_token(self, mock_decode):
                 'message': UserViewSet.MISSING_USER_INFO
             }
         )
+
+
+class CustomIngredientsApi(TestCase):
+    unit1 = None
+    unit2 = None
+
+    def setUp(self):
+        """
+        Sets up a test database with test values
+        """
+        self.user1 = User.objects.create(
+            username=USER_VALID_TOKEN_PAYLOAD.get('sub'),
+            email=USER_VALID_TOKEN_PAYLOAD.get(CUPBOARD_EMAIL_CLAIM)
+        )
+
+        self.ing1 = CustomIngredient.objects.create(
+            user=self.user1,
+            name='test_ingredient1',
+            type='test_type1'
+        )
+        self.ing2 = CustomIngredient.objects.create(
+            user=self.user1,
+            name='test_ingredient2',
+            type='test_type1'
+        )
+
```

Comment:
Maybe create a `self.cust_ing` dictionary with the custom ingredient name and type set. This way you can just change this in one place instead of writing beef and meat in several places. Just do a get on this dictionary instead.